index.js

```diff
@@ -127,7 +129,7 @@ export function execa(file, args, options) {
 				escapedCommand,
 				parsed,
 				timedOut,
-				isCanceled: context.isCanceled,
```

Comment:
Should `isCanceled()` be directly inlined there?

```suggestion
				isCanceled: context.isCanceled || (parsed.options.signal ? parsed.options.signal.aborted : false);
```